templates/downloads/supernav.html

```diff
@@ -11,14 +11,16 @@
             <a class="button" href="{{ data.python3.url }}">{{ data.python3.release.name }}</a>
             <a class="button" href="{{ data.python2.url }}">{{ data.python2.release.name }}</a>
         </p>
-        <p>Not the OS you are looking for? Python can be used on 21 different operating systems and environments. <a href="{% url 'download:download' %}">View the full list</a>.</p>
+        <p>{{navigator.appVersion}}</p>
+        {% if data.os.slug == 'windows' and data.python3.release.name >= 'Python 3.5' %}<p><b>NB Python 3.5+ <i>cannot</i> be used on WinXP or earlier</b></p>{% endif %}
```

Comment:
b and i tags are deprecated. Can we change them with strong and em respectively?